safe_transaction_service/tokens/services/price_service.py

```diff
@@ -194,14 +197,26 @@ def get_kcs_usd_price(self) -> float:
         except CannotGetPrice:
             return self.coingecko_client.get_kcs_usd_price()
 
-    @cachedmethod(cache=operator.attrgetter("cache_eth_price"))
+    @cachedmethod(cache=operator.attrgetter("cache_ether_usd_price"))
     @cache_memoize(60 * 30, prefix="balances-get_eth_usd_price")  # 30 minutes
+    def get_ether_usd_price(self) -> float:
```

Comment:
Could you add a test for ether USD like we do for native coin?

Reply:
This method is already tested on the native coin one, but sounds good, I will

Reply:
Done